articles/synapse_docker_registry.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Synapse Docker Registry
+layout: article
+excerpt: The basics of using the synapse docker registry
+---
+
+##Synapse Docker Registry
+Synapse now has its own docker registry which will allow you to have private docker repositories.  Please click [here](https://www.docker.com/products/overview) to learn how to use docker for mac, windows or linux. Below is a brief tutorial on how to use the synapse docker registry using the command line.
+
+### Step 1. Check out or build some images
+**Check out some images**
+
+```
+docker pull ubuntu
+```
+
+**Build your own image from a Dockerfile**
+
+```
+docker build -t  docker.synapse.org/syn12345/my-repo path/to/dockerfile
+```
+
+###Step 2. View your images
```

Comment:
Combine these docker specific steps into a single step.  Something to the effect.  Lets assume we are building a custom docker image (see [documentation] on using docker)...Bla bla blaa

Then focus on the details of interacting with Synapse to store the image.